### ISSUE TITLE
SetValue Actionlet throws an NPE

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#18468]
When using the setValue Actionlet, it throws an NPE.  This is because it it passing in a null URI and the `MockHttpRequest` now expects a uri value.

**MockHttpRequest**
https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/mock/request/MockHttpRequest.java#L39

**SetValueActionlet**
https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SetValueActionlet.java#L72

```
14:36:23.653  WARN  web.ContentletWebAPIImpl - com.dotmarketing.exception.DotRuntimeException
14:36:23.677  ERROR ajax.ContentletAjax - null
14:36:24.034  INFO  timedcache.TimedCacheProvider - ***	 Building Cache : esquerycache, size:10000, seconds:60,Concurrency:32 (every 60000 millis)
14:37:03.515  ERROR ajax.BrowserAjax - null
com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException: null
	at com.dotmarketing.portlets.workflows.actionlet.SetValueActionlet.executePreAction(SetValueActionlet.java:111) ~[classes/:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPreCheckin(WorkflowAPIImpl.java:2285) ~[classes/:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPreCheckin_aroundBody164(WorkflowAPIImpl.java:2268) ~[classes/:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl$AjcClosure165.run(WorkflowAPIImpl.java:1) ~[classes/:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[classes/:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:59) ~[classes/:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[classes/:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[classes/:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowPreCheckin(WorkflowAPIImpl.java:2267) ~[classes/:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl.fireWorkflowNoCheckin_aroundBody186(WorkflowAPIImpl.java:3179) ~[classes/:?]
	at com.dotmarketing.portlets.workflows.business.WorkflowAPIImpl$AjcClosure187.run(WorkflowAPIImpl.java:1) ~[classes/:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[classes/:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:59) ~[classes/:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[classes/:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[classes/:?]
```

**Comment by wezell** [comment]
PR: #18469 

To test, load the old starter and try to use the "archive" actionlet on a blog entry.

**Comment by fabrizzio-dotCMS** [comment]
This issue was tested using branch `release-5.3.0` and starter `20200429` 
I did what is suggested above to test the issue. I was able to verify that the code that uses the mock request is invoked by setting a breakpoint. 

I used one of the existing blogs that came with the starter.  I archived it and verified that the code in review was called.

I also tried archiving a brand new blog. I also used the bulk actions to do the same. 
I was able to verify the NPE is gone for good.
